source/web/src/routes/Configure.vue

```diff
@@ -45,44 +57,59 @@ export default {
   data () {
     return {
         processing: false,
-        netinfo: null,
-        form: {
-            mountTarget: null
-        },
-        selected: null
+        uid: '1000',
+        gid: '1000',
+        path: '/efs'
     }
   },
-  created: function () {
-      this.getFilesystemNetinfo()
-  },
   computed: {
-      options: function () {
-          let mountTargetNames = Object.keys(this.netinfo)
-          return mountTargetNames
+      valid() {
+          return this.uid != "" && this.gid != "" && this.path != "" && this.path.charAt(0) == '/'
       }
   },
   methods: {
-      onSubmit(evt) {
-        evt.preventDefault()
-        let mountTarget = this.selected
-        let mountTargetNetinfo = this.netinfo[mountTarget]
-        this.createManagerLambda(mountTargetNetinfo)
+      async onSubmit(evt) {
+        evt.preventDefault();
+
+        if (this.valid) {
+            let mountTargetNetinfo = await this.getFilesystemNetinfo()
+        
+            mountTargetNetinfo['uid'] = this.uid
+            mountTargetNetinfo['gid'] = this.gid
+            mountTargetNetinfo['path'] = this.path
+
+            console.log(mountTargetNetinfo)
+
+            this.createManagerLambda(mountTargetNetinfo)
+        }
+        else {
+            alert("Form Validation Error. Check the form input and try again.")
+        }
       },
       formatNetinfo (netinfo) {
-          let tmpNetinfo = {}
+          let subnets = []
+          let securityGroups = []
+        
           for (var i=0, n=netinfo.length; i < n; ++i ) {
-              let netinfoKeys = Object.keys(netinfo[i])
-              let mountTargetName = netinfoKeys[0]
-              tmpNetinfo[mountTargetName] = netinfo[i][mountTargetName]
-              
+              if (i === 5) { break; }
```

Comment:
Why does the variable i have to equal 5. Just curious.

Reply:
I discovered that there is a maximum number (5) of mount targets that a lambda can be connected to.

Reply:
👍